Managers/FrameMan.h

```diff
@@ -626,12 +622,10 @@ namespace RTE {
 
 	private:
 
-		static const std::string c_ClassName; //!< The friendly-formatted type name of this object.
-
 		/// <summary>
 		/// Enumeration with different settings for the SaveBitmap() method.
 		/// </summary>
-		enum SaveBitmapMode { SingleBitmap = 0, ScreenDump, WorldDump, ScenePreviewDump};
+		enum SaveBitmapMode { SingleBitmap = 0, ScreenDump, WorldDump, ScenePreviewDump };
```

Comment:
I dunno.
Generally I think that ones that you aren't going to constantly deal with and/or have little entries and/or have no explicitly defined values can be onelined.
Since the first entry always defaults to 0 (unless you explicitly define otherwise) and the rest increment by default i'm removing the `= 0` in places I remember anyway.
You wouldn't look at this enum if it weren't for this little change to the line. It's private, it's very specific so you're not going to encounter it outside the very specific code it's used it so it doesn't even need to catch our eye.

Reply:
These are fair arguments. I don't like it visually, I find it makes it less visually clear to me that it's an enum. But if it's private and infrequently used then I guess I can live with it in an "out of site, out of mind" sort of way.